pkg/internal/cnsoperator/cnsfilevolumeclient/cnsfilevolumeclient.go

```diff
@@ -35,6 +35,9 @@ import (
 // FileVolumeClient exposes an interface to support
 // configuration of CNS file volume ACL's.
 type FileVolumeClient interface {
+	// GetClientVmsFromIPList returns the list of client vms associated with a given External IP address
```

Comment:
Can you align the paragraph width with the other function comments for this interface?

Reply:
Done